packages/cli-plugin-metro/src/tools/loadMetroConfig.ts

```diff
@@ -14,9 +14,14 @@ const INTERNAL_CALLSITES_REGEX = new RegExp(
     '/Libraries/YellowBox/.+\\.js$',
```

Comment:
I wonder if all of these should be prefixed with `react-native`. (Not sure)

Reply:
Yeah probably -- I opted to minimize the diff because I wasn't sure why it was this way.

Reply:
@rickhanlonii mind following up with a PR with adjusted paths?

Reply:
The reason the `react-native` prefix is missing is because forks of React Native such as [`react-native-tvos`](https://github.com/react-native-tvos/react-native-tvos) (and `react-native-windows`?) have copies of these files but are installed within `node_modules` using their forked name. If you match on `react-native`, you'll break `YellowBox` for all forks. You might want to use `node_modules\/(react-native(?:-(.*?))?)\/` here and below where you added the match on `/node_modules/react-native/index.js`.

Reply:
A wild Nakazawa appears!

Christoph used Context.

It's super effective!